src/dc_imex.rs

```diff
@@ -793,7 +791,7 @@ unsafe fn export_backup(context: &Context, dir: *const libc::c_char) -> libc::c_
                                         let mut processed_files_cnt = 0;
                                         for entry in dir_handle {
                                             if entry.is_err() {
-                                                current_block = 2631791190359682872;
+                                                ok_to_continue = true;
```

Comment:
i guess ``ok_to_continue = false`` is correct here.

Reply:
the C-source also does this, it seems (silently breaking the loop of copying files and returning success=true) so it' snot an issue of this PR here. maybe i am missing something? cc @r10s